.gitignore

```diff
@@ -8,3 +8,6 @@ compiler/**/.merlin
 legifrance_oauth*
 *.html
 .vscode/
+.ninja_*
```

Comment:
> So here I guess you simply create a `ninja.build` in the same folder where you launch `clerk`.

Yes, indeed, it's the current behavior.

> But since you re-generate the ninja file each time, wouldn't it be better to hide it in the `/tmp` folder?

If Clerk doesn't use previous generated `build.ninja` files yet, the user can however call by himself or herself `ninja test` -- or other sub-build statements as shown in the [screenshot](https://github.com/CatalaLang/catala/pull/186#issue-1119720964). But this will assume a knowledge of the structure of the generated `build.ninja` file -- which could be added in the README file by the way.
In any case, adding a flag to specify the output directory for the `build.ninja` file could be useful -- if not specified, the file will be generated in `/tmp`. Is it fine for you?

Reply:
Yep fine with me!

Reply:
There is an issue with the `-o` flag, indeed, paths inside `build.ninja` are relative to its directory. Therefore, paths must be absolute inside the `build.ninja` file. However, this leads to error when diffing bad output test files such as:
```
--- /home/emile/Projects/ocaml/catala/tests/test_array/bad/output/fold_error.catala_en.A.Interpret	2022-02-22 16:23:05.632574785 +0100
+++ -	2022-02-24 23:39:29.391176114 +0100
@@ -3,21 +3,21 @@
 --> integer
 
 Error coming from typechecking the following expression:
-  --> tests/test_array/bad/fold_error.catala_en
+  --> /home/emile/Projects/ocaml/catala/tests/test_array/bad/fold_error.catala_en
    | 
 10 |   definition list_high_count equals number for m in list of (m >=$ $7)
    |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    + Article
```
Should we assume that having a build.ninja file in the git tree isn't that bad?

Reply:
Ah indeed, I guess in that case you have to keep both the absolute and relative paths: give the absolute path to `build.ninja` and keep the relative path for calling the Catala compiler binary.

Reply:
I retried and using relative paths inside the `build.ninja` file now works.. :no_mouth: